docs/usage.md

```diff
@@ -5,90 +5,225 @@
+This scenario generates following alerts, each at an interval of 2 seconds:
 
-4. **OT - Add Sample Asset Change Activity Record**
+- Stuxnet peer-to-peer communication attempt
+- Anomalous communication between two Windows XP machines
+- Communication between public and private networks
+- STEP7 configuration download command
 
-- Browse to `Simulations` > `OT - Add Sample Asset Change Activity Record` scenario and click **Simulate Scenario**.
+To run this scenario and generate the alerts, perform the following steps:
 
-    >**NOTE**: Playbook: **Scenario - OT - Asset Change Activity** has variable `dueDays` (*Days required to complete change activity*). You can populate these variables with your preferred values in the *Configuration* step.
+1. Under the FortiSOAR&trade; menu, select to open **Simulations**.
 
-- With a given value of due date in `dueDays` two Asset Change Activity entries are made.
-    - Add New Cyber Asset - Siemens S7-400-S621
-    - Medium Impact Baseline Change - Stratix 5950-RW41
+2. Select the scenario **OT - Stuxnet Attack**.
 
-- Assets are linked and can be seen in the Assets & Vulnerabilities tab.
+3. Click **Simulate Scenario**.
 
-- Once the record has been created, execute the **Add Task From Templates** to assign different tasks to different people. 
+4. To view the added sample alerts, under the FortiSOAR&trade; menu select to open **Incident Response** > **Alerts**.
 
-- New tasks get created as priority `Medium` once the previous task is completed.
+![](./res/added-stuxnet-alerts.png)
 
-- New task can be added by using **Add New Task** playbook.
+### Enrichment
 
-- Once all task get completed record get closed.
+- Each alert is linked to the following assets based on their source and destination IP:
 
-- To generate the report, execute **Generate Compliance Report** and report get attached to particular record comments.
\ No newline at end of file
+    - PLC S7-400-324
+    - Dell - 5490-42E6
+    - ThinkCentre M910s-1181
+    - HP - PB7181
+
+    ![](./res/stuxnet-peer-to-peer-linked-assets.png)
+
+- The *Technique ID* in an alert ensures linking of MITRE ATT&CK's data &ndash; *Group*, *Tactics*, *Technique*, *Sub-Technique*, and *Software*. This data appears under the alert's **Correlations** tab.
+
+    ![](./res/alert-mitre-correlations-tab.png)
+
+- Mitre ATT&CK's suggested *Mitigations* are fetched using the *Technique ID* and are listed under the alert's **Recommended ATT&CK Mitigations** tab.
+
+    ![](./res/alert-mitre-mitigation-tab.png)
+
+<table>
+    <tr>
+        <td><strong>TIP</strong></td>
+        <td>The <strong>Workspace</strong> panel lists alerts similar to <strong>Stuxnet peer-to-peer communication attempt</strong>. To know more about the Workspace panel refer to the platform documentation at <a href="https://docs.fortinet.com/document/fortisoar/7.4.1/user-guide/841633/war-rooms#Workspace_-_Enabling_Communication">Workspace - Enabling Communication</a>.</td>

```

Comment:
link should be -- https://docs.fortinet.com/document/fortisoar/7.4.1/user-guide/841633/war-rooms#Workspace_-_Enabling_Communication

Reply:
Link or link text? Link is same...